documentation/api/guides/deploy.md

```diff
@@ -21,10 +21,11 @@ For more information on how deployments work, please see the [general deployment
    and verify the SHA is the same as the value you deployed.
 1. Review and Approve the automatically-generated changelog pull request in the
    repository.
-1. In the event of errors or problems, repeat the deployment process using the
-   latest stable version of the application. You can find the release version
-   number in the [changelogs](/changelogs/index), and then the tag to pass to
-   the action is the version number prefixed with "rel-", for example
-   "rel-2023.07.03.17.52.00".
+1. In the event of errors or problems, rollback the application by running the
+   appropriate deployment workflow from the WordPress/openverse-infrastructure
+   repository using the tag of the latest stable version. You can find the
+   release version number in the [changelogs](/changelogs/index), and then the
+   tag to pass to the workflow is the version number prefixed with "rel-", for
+   example "rel-2023.07.03.17.52.00".
```

Comment:
Is this accurate (here and in the other applications)? Is the appropriate tag the `rel-` tag, or the tag obtained from the version endpoint?

Reply:
It is accurate. Here's a screenshot of the latest API releases on https://github.com/wordpress/openverse/pkgs/container/openverse-api:
 
![Screenshot from 2024-03-05 19-02-05](https://github.com/WordPress/openverse/assets/6351754/832c0133-92dc-4c62-beb4-7afca4b5a40c)

I believe with the way our builds work that those two latest images are essentially the same image, or put another way the `rel-` tagged release is a duplicate of the immediate previous release.

Reply:
Yes, the `rel-` tag. We could probably drop that if we wanted to, one day, and just use the date without the `rel-` prefix. That is probably less confusing than needing to swap the application slug for `rel-`? Instead, we'd just drop the application slug?

Reply:
No strong feelings either way -- I think we see/use the `rel-` tags plenty when deploying catalog/ingestion-server so it's not especially confusing as is. I just wanted to double-check :)